pandas/tests/indexes/test_frozen.py

```diff
@@ -10,7 +10,7 @@ class TestFrozenList(CheckImmutable, CheckStringMixin):
     mutable_methods = ('extend', 'pop', 'remove', 'insert')
     unicode_container = FrozenList([u("\u05d0"), u("\u05d1"), "c"])
 
-    def setup_method(self, method):
```

Comment:
can you fixture here

Reply:
What exactly are we fixturing here?

Reply:
everything in setup_method (to remove it), if its not crazy. also ok to do later.

Reply:
I realize that fixtures are more idiomatic `pytest`, but since those attributes are not being used outside of the test class, I'm not sure why we would need to fixture them...

It's not a super big deal.  But just wanted to understand the rationale.

Reply:
oh, setup_method is the older way of doing this (it might be deprecated) its just more consistent with all other parts of the testing suite.

Reply:
Not deprecated AFAIK.  It's just more consistent with the `unittest` library format.  I think we can leave for now, given that we have other test classes in that file.

Reply:
well, that's the think we are aggressively moving to pytest methodologies everywhere else. as I said can do in a followup if you want.

Reply:
Ah, fair enough.  Let's do in a follow-up then.

Reply:
I should also point out that there is still a good amount of inheritance / OOP in the test classes here, so making it purely functional is not straightforward.